### PR TITLE
Fixed footer ad format for all themes

### PR DESCRIPTION
**Pull request by davidfischer**
Goals:

- Let people know about the new ad format rollout and timeline.
- Get commercial companies using RTD community to support RTD (by moving to .com or otherwise)

One thing I would like to get down a little better is our exact upsell path for companies who don't want ads. I feel like we are close there but I think we could use a little refinement. Publishing the blog probably won't immediately change anything but the moment community and house ads start showing up on custom themes, we are going to get inquiries.

### CODE REVIEW

File: fixed-footer-ad-all-themes.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+New Ad Format Coming to Read the Docs Community Sites
+=====================================================
+
+We view our ad program as a way to keep Read the Docs itself sustainable,
+and to use it to better support the community.
+Advertising has allowed us to have full-time employees adding new features
+and responding to issues in our issue tracker
+and to share thousands of dollars with the open source community
+via our revenue share program and grants.
```
Comment (ericholscher) [inline fixed-footer-ad-all-themes.rst]: bit of a run-on sentence. Can probably be 2.

File: fixed-footer-ad-all-themes.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Currently, about 30% of our site traffic does not have any advertising.
+When we first launched ethical advertising in 2016,
+we launched only on :doc:`specific documentation themes <ads-on-other-themes>`.
+We purposely did this slowly to make sure our ads look integrated
+with Read the Docs and less obtrusive to users.
+
+For a while, this wasn't a problem because we had not reached capacity with advertising.
+We did not have enough paid advertisers to support the amount of pageviews Read the Docs served.
+Fast forward a couple years, and our advertising model has proven to be successful.
```
Comment (ericholscher) [inline fixed-footer-ad-all-themes.rst]: Link to the other post here?

File: fixed-footer-ad-all-themes.rst [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+* To start with, we are going to enable this ad format for community and house advertising
+  on pages that already have ads on them.
+  As always, we never display two ads on the same page so this ad will be instead of any other ad.
+* After that, we will allow paid advertisers to use this new format.
+  Again, we will start only on pages that already have ads.
+* We will email users of custom documentation themes to let them know this change is coming.
+* At first, we will show only community and house ads on these custom themes.
+* Finally, we'll turn on our normal ad rotation to all themes.
+  We expect to complete this in August.
```
Comment (ericholscher) [inline fixed-footer-ad-all-themes.rst]: I'd say July. Better to be late than early :)

**Comment by davidfischer** [comment]
I made those updates. I tentatively put the date for next Tuesday but if we feel good enough to get this out before then, I'm ok with that.

**Pull request merged**